google/ggrc-core
# [WIP] GGRC-7093 FE: enable markdown in Risks

# Issue description

Markdown in Risks isn't supported

# Steps to test the changes

1. Insert some markdown in description field into risks table in the database
2. Open changed risk in the app
3. Verify that markdown is rendered as expected and works

# Solution description

There's already a fine implementation of making any object supports markdown. It's done in https://github.com/google/ggrc-core/commit/ee906c8a1c546e6ecf05b46662922ede6bd7aa8b and https://github.com/google/ggrc-core/commit/105b925effd6d90b489471278b21858e54369e61. So all I have to do is apply changeableExternally mixin to Risk object.

# Sanity checklist


# PR Review checklist
